### ISSUE TITLE
Translations clipped 

### ISSUE THREAD

**Issue by Perezmarc** [Animenosekai/translate#6]
Hi, 
You have something awesome going here :)
I have troubles with wrong spacing of translations, and others with googletrans and py-googletrans, so I gave yours a try.

The translations are clipped for some reason:
Source:
```
NYC Hot Sauce Expo NYC Hot Sauce Expo: Fiery Food & More Brooklyn Expo Center (Brooklyn, NY) This two-day tribute to all things hot and spicy blazes its way into the Brooklyn Event Center for the...
```

```
translatorpy ca: NYC Hot Sauce Expo NYC Hot Sauce Expo: menjar ardent
translatorpy es: NYC Hot Sauce Expo NYC Hot Sauce Expo: comida ardiente
translatorpy en: NYC Hot Sauce Expo NYC Hot Sauce Expo: Fiery Food
```

```
Google Translate API ca : NYC Hot Sauce Expo NYC Hot Sauce Expo: Fiery Food & More Brooklyn Expo Center (Brooklyn, NY) Aquest homenatge de dos dies a totes les coses picants i calentes s’obre al Centre d’esdeveniments de Brooklyn per ...
Google Translate API es : NYC Hot Sauce Expo NYC Hot Sauce Expo: Fiery Food & More Brooklyn Expo Center (Brooklyn, NY) Este homenaje de dos días a todo lo picante y picante se abre camino en el Brooklyn Event Center para el ...
Google Translate API en : NYC Hot Sauce Expo NYC Hot Sauce Expo: Fiery Food & More Brooklyn Expo Center (Brooklyn, NY) This two-day tribute to all things hot and spicy blazes its way into the Brooklyn Event Center for the...
```

**Comment by Animenosekai** [comment]
I think that I mis-encoded the `&` sign and it is thinking that it's a new parameter.

**Comment by Animenosekai** [comment]
@Perezmarc The bug has been fixed in v1.5

Please update your `translatepy` installation to v1.5 with `pip install --upgrade translatepy`
